src/gui/appclient.cpp

```diff
@@ -639,6 +648,30 @@ bool AppClient::connectToServer() {
     return true;
 }
 
+void AppClient::updateSentryUser() const {
```

Comment:
The current user can be obtained with _gui->currentUserDbId()
(use the cache instead of executing a C/S request)

Reply:
Thank for this suggestion, here is the implementation: [3390f1a](https://github.com/Infomaniak/desktop-kDrive/pull/274/commits/3390f1a3fa5caa30cefbe15d0825c46fb79bcd57)